src/Uno.UI/UI/Xaml/Controls/TextBlock/JavaStringCache.Android.cs

```diff
@@ -3,33 +3,33 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
-using System.Threading;
-using System.Text;
-using Windows.Foundation;
 
 using Uno;
-using Uno.Extensions;
-using Uno.UI;
 using Uno.Foundation.Logging;
-using Microsoft.UI.Xaml.Media;
-using Uno.Collections;
-using Android.Security.Keystore;
-using Java.Security;
 using Uno.Buffers;
 using Windows.System;
 
 namespace Microsoft.UI.Xaml.Controls
 {
 	/// <summary>
-	/// A TextBlock measure cache for non-formatted text.
+	/// A cache for native java strings. This cache periodically evicts entries that haven't been used
+	/// in a while. Additionally, it also evicts the least recently used entries when adding new entries beyond a certain
+	/// capacity. Limiting the total capacity is necessary to
 	/// </summary>
 	internal static class JavaStringCache
```

Comment:
Can you add a feature configuration flag that allows for disabling the cache altogether? `IsJavaStringCachedEnabled`?